content/_index.md

```diff
@@ -6,7 +6,17 @@ The teams that have solved the most puzzles at the end of the contest qualify fo
 
 The BAPC 2021 is hosted by [Vrije Universiteit Amsterdam](https://www.vu.nl/), and organized by a committee of [study association STORM](https://storm.vu).
 
-There is an online contest on [Kattis](https://bapc21online.kattis.com) 13:00-18:00CET on Sunday October 31st, and problems will remain available on Kattis afterwards.
+## Contest materials
+
+- [BAPC problems](/bapc2021-problems.pdf)
+- [BAPC solutions](/bapc2021-solutions.pdf)
+- [BAPC testdata and solutions (162MB)](https://commissies.ch.tudelft.nl/chipcie/archief/2021/bapc/solutions.zip)
+- [preliminaries problems](/prelims2021-problems.pdf)
+- [preliminaries solutions](/prelims2021-solutions.pdf)
+- [preliminaries testdata and solutions (70MB)](https://commissies.ch.tudelft.nl/chipcie/archief/2021/dapc/solutions.zip)
```

Comment:
I rather host everything at github and think about a good way of how we should mirror this when we have a non-profit meeting again. As to me this looks like creating the DNS problem again.